ota-plus-web/app/reactapp/src/components/updates/createWizard/Step1.jsx

```diff
@@ -93,6 +96,7 @@ class Step1 extends Component {
                 })
               )}
             </div>
+            <div className="information-text">{t('updates.creating.wizard.information_step1')}</div>
```

Comment:
Where we are checking that user has only one ECU?
Please check the case with many ECUs. This text will be scrolled and placed in the middle of the list.